src/modules/localStorage.js

```diff
@@ -0,0 +1,45 @@
+import { addTask } from './index.js';
+
+export const saveData = (taskData, arr) => {
+  if (localStorage.getItem('ObjectSaved') == null) localStorage.setItem('ObjectSaved', '[]');;
+  const savedData = JSON.parse(localStorage.getItem('ObjectSaved'));
+  if (savedData.length < arr.length) savedData.push(taskData);
+  localStorage.setItem('ObjectSaved', JSON.stringify(savedData));
+};
+
+export const removeData = (index) => {
+  if (localStorage.getItem('ObjectSaved') == null) return;
+  const savedData = JSON.parse(localStorage.getItem('ObjectSaved'));
+  savedData.splice(index, 1);
+  localStorage.setItem('ObjectSaved', JSON.stringify(savedData));
+};
+
+export const updateInput = (object) => {
+  if (localStorage.getItem('ObjectSaved') == null) return;
+  const savedData = JSON.parse(localStorage.getItem('ObjectSaved'));
+  const OldObject = savedData[object.index];
+  OldObject.description = object.description;
+  localStorage.setItem('ObjectSaved', JSON.stringify(savedData));
+};
+
+export const updateIndex = (arr) => {
+  if (localStorage.getItem('ObjectSaved') == null) return;
+  const savedData = JSON.parse(localStorage.getItem('ObjectSaved'));
+  arr.forEach( (element, ind) => {
+    element.index = ind;
+  });
+  for (let i = 0; i < arr.length; i++){
+    savedData[i].index = i;
+  };
+  localStorage.setItem('ObjectSaved', JSON.stringify(savedData));
+};
```

Comment:
You did really well with the `addTask` function of your Todo list,weldone 👍 . However , for your Todo list, editing a task and deleting a task doesn't seem to work as expected. You can only select a task but you cant edit and delete it which is a major part of the project requirement. Please kindly work on these functions so as to have a better project that achieves it's major goals 😄 

Project requirement :

![image](https://github.com/EvansSnave/to-do-list/assets/20537712/23be6257-317d-40e9-ba1b-07b4070cfa20)

your project current behaviour :

https://github.com/EvansSnave/to-do-list/assets/20537712/2636c834-043a-4d37-a81e-fb74465e3880

Reply:
@usorfaitheloho hello! You can edit the description if you click on the task description and you can delete tasks if you click on the icon

Reply:
@EvansSnave  you can only highlight the task to be deleted when you click on the icon but it doesnt delete. I clicked on the task description and it doesn't edit

Reply:
@usorfaitheloho Really? I thought I did it right look for me it works

https://github.com/EvansSnave/to-do-list/assets/127534196/b086876f-ce34-4a9c-946e-cbb8f8478ece